ql/src/java/org/apache/hadoop/hive/ql/cache/results/QueryResultsCache.java

```diff
@@ -651,7 +652,7 @@ public void notifyTableChanged(String dbName, String tableName, long updateTime)
     List<CacheEntry> entriesToInvalidate = null;
     rwLock.writeLock().lock();
     try {
-      String key = (dbName.toLowerCase() + "." + tableName.toLowerCase());
+      String key = TableName.getDbTable(dbName, tableName).toLowerCase();
```

Comment:
I would think that at this point we should already have a tablename from somewhere; constructing it here doesn't really make sense